06 piškvorky/piskvorky1D.py

```diff
@@ -0,0 +1,77 @@
+from random import randrange
+import time
+
+# TICK-TACK-TOE
+
+
+def score(area):  # fce get list(area) and return result in accordance with area
+
+    # draw
+    if '-' not in area:
+        print(' "!" ')
+        quit()
+    # players won
+    elif (3 * player_symbol) in area:
+        print(' "X" ')
+        quit()
+    # pc won
+    elif (3 * pc_symbol) in area:
+        print('"O"')
+        quit()
+    # next play
+    else:
+        print('"-"')
+
+
+def move(area, position, symbol):  # put symbol into area, get area, position 1 - 19, symbol
+
+    area = list(area)
+    area[position - 1] = symbol
+    area = ''.join(area)
+    return area
+
+
+def player_move(area):  # get input(position) from player, test input and return area with symbol
+
+    position = input('Kam umístit X? Vyber políčko 1 - 20. Tvá volba: ')
+    while True:
+
+        position = str(position)
+        if position.isalpha():
```

Comment:
Čo ak užívateľ zadá znak? Nebolo by lepšie skontrolovať či vstup nie je číslo? String má na to svoju metódu (obdoba tejto). Pokiaľ zistíš že je to čokoľvek iné ako číslo tak mu napíšeš hlášku a vrátiš sa na začiatok while aby si sa spýtala znova.

Reply:
to by šlo :)